### DISCUSSION TITLE
What does this error mean?

### DISCUSSION THREAD

**Discussion by Jiayou-Chao** [plandex-ai/plandex#141] [Q&A]
I think we really need a better document for error messages. I have been receiving error messages like: 
 🏗  Built plan                                                                                   
📄 new_claude_script.py ✅                                                                       

🚨 Server error
  → Stream error
    → EOF

and 

🚨 Server error
  → Stream error
    → Stream finished with no choices

I don't really understand what happened, but I can see API usage in my account. I tried both ` Built-in | gpt-4o-latest` and `Built-in | gpt-4-turbo-latest`.

Besides, it never worked when I set all models to be gpt-3.5-turbo. It just said 'I will do ...' and then stopped and didn't change anything. Does anyone know why?

**Comment by danenania** [comment]
Hey @Jiayou-Chao is this with cloud or self-hosted?

**Comment by danenania** [comment]
3.5-turbo generally isn't capable enough to follow the instructions for the planner/builder roles unfortunately.
